app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -40,6 +40,10 @@ def self.default_blacklisted_event_names
     )
   end
 
+  def self.metrics_collect_queue_name
+    MetricsCollectorWorker.default_queue_name
```

Comment:
I tried this locally in rails console and `ManageIQ::Providers::Microsoft::InfraManager.metrics_collector_queue_name` just returns nil.

I think that's safe.  Otherwise, we could have the impl in the base `MetricsCollectorWorker` return empty string instead to avoid NPEs.